controllers/gamesC.js

```diff
@@ -1,7 +1,13 @@
-const { getCategoryList } = require("../models/gamesM.js");
+const { getCategoryList, getUserList } = require("../models/gamesM.js");
 
 exports.getCategories = (req, res) => {
   return getCategoryList().then((data) => {
     res.status(200).send(data);
   });
 };
+
+exports.getUsers = (req, res) => {
+  return getUserList().then((data) => {
+    res.status(200).send(data);
```

Comment:
response needs to be sent back as an object, with a key of users.
`data` could also be named slightly better to reflect the content